### DISCUSSION TITLE
agent-handbook/docs/agent-intro/tool-usage

### DISCUSSION THREAD

**Discussion by giscus** [wj-Mcat/agent-handbook#6] [General]
# agent-handbook/docs/agent-intro/tool-usage

大多数的工具调用基于 ReAct 模板，可以让LLM 与外部工具进行交互，从而从一定程度上缓解幻觉问题、提升能力边界。

https://wj-mcat.github.io/agent-handbook/docs/agent-intro/tool-usage

**Comment by wj-Mcat** [comment]
test
